dotcom-rendering/src/web/components/CalloutEmbed/Form.test.tsx

```diff
@@ -246,7 +246,8 @@ describe('Callout from', () => {
 		const input = screen.getByTestId<HTMLInputElement>(
 			`form-field-${fileField.id}`,
 		);
-		user.upload(input, file);
+
+		await user.upload(input, file);
```

Comment:
Not sure if this function was synchronous before and is now asynchronous, but the test won't pass now testing-library has been updated unless we await for the upload t complete